packages/cli/src/constants.ts

```diff
@@ -16,6 +16,12 @@ export const RootFolderNode = new QTreeNode({
   default: "./",
 });
 
+export const EnvNode = new QTreeNode({
+  type: "text",
+  name: "env",
+  title: "Select or create an env for the project",
```

Comment:
Do we allow creating new env for deploy stage?
In other words, do we need to distinguish `select only` with `select or create`?

Reply:
For now, if user uses a non-existent environment, CLI will create an env and pop up an error. @qinezh what's your opinion?
![image](https://user-images.githubusercontent.com/13211513/129533157-9918f62c-8af6-4aba-9518-59a532d62218.png)

Reply:
for this demo, how about throwing a friendly user error like `the specified env xxx does not exist.`

Reply:
Fixed.